packages/clima_ui/lib/screens/weather_screen.dart

```diff
@@ -77,15 +104,17 @@ class WeatherScreen extends HookWidget {
             return;
           }
 
-          isLoading.value = true;
+          isCityChanging.value = true;
           await cityStateNotifier.setCity(City(name: trimmedCityName));
           if (context.read(c.cityStateNotifierProvider) is! c.Error) {
-            await load();
+            await fullWeatherStateNotifier.loadFullWeather();
           }
-          isLoading.value = false;
+          isCityChanging.value = false;
         },
         title: Text(
-          'Updated ${DateFormat.Md().add_jm().format(DateTime.now())}',
+          fullWeatherState.fullWeather == null
+              ? ''
+              : 'Updated ${DateFormat.Md().add_jm().format(DateTime.now())}',
```

Comment:
Can't we just pass `null` as `title` instead of passing an empty string to `Text`?

Reply:
It didn't work. It will display the "Enter city name" text if null